### ISSUE TITLE
Using for Arduino UNO

### ISSUE THREAD

**Issue by Latsbben** [felixgalindo/HPMA115S0#1]
Hi! I am a new to Arduino. I notice that you in your example is using a feather. Will I have to change anything to use it for Arduino UNO, e.g. the reference to the RX and TX pins?

**Comment by felixgalindo** [comment]
The example code uses a software serial so I believe you shouldn't have to change the pins on an Arduino UNO. Try it out and let me know if it works.

**Comment by Latsbben** [comment]
I connected the RX and TX to the RX and TX pins on the Arduino UNO Board. I get some error messages when I am starting the program, e.g. 

avrdude: stk500_getsync() attempt 1 of 10: not in sync: resp=0x00

These error messages continue also for when I am loading other sketches, e.g. Blink. But if I remove the TX connection the error during loading disappear.

In the serial monitor I first got some gibberish at baud 9600 but was able to get readable text by changing to baud 57600 in the monitor.

When the HPM is connected through the RX and TX, it writes the following:
PS- Reading Particle Measurements...
PS- Sending cmd 68 1 4 93
PS- Waiting for cmd resp
And then repeats this endlessly

To me it looks like it is sending through the RX but is not getting anything back from the sensor through the TX. I have installed the newest FTDI drivers and now I am not sure what to do next. Any wise tips?

**Comment by felixgalindo** [comment]
Which pin #'s are connected to the sensor? It looks like you have the sensor connected to the debug serial output which has a baud of 57600. You should have it connected like this: Pin10->SensorRX, Pin11->SensorTx. The baud rate on these pins are 9600.Also, make sure your sensor is powered by 5v sharing ground with the UNO of course.

**Comment by Latsbben** [comment]
Thanks for the advice! And apologies for the slow turn-around; I have been off the grid for some days. The program now uploads without any error messages. I have tried to connect the wires from the sensor both to pin 10&11 and pin RX&TX on the board. But still get the same error messages for both cases, e.g. 

PS- Reading Particle Measurements...
PS- Sending cmd 68 1 4 93
PS- Waiting for cmd resp

If I change the Baud from 57600 to 9600 the writing on the serial monitor becomes gibberish, in both cases.

I have a separate 5V supply for the sensor to ensure that the voltage is right, so do not think that is causing the problems.

I have also tried to disconnect the sensor from 3.3 V, but that does not change anything. 

If you have any other advice, it is much appreciated. And apologies for the repeated questions, as I wrote I am completely new to Arduino and the only way to learn is to ask :)
